distributed-map/backup/src/main/java/Cluster.java

```diff
@@ -0,0 +1,54 @@
+import java.lang.management.ManagementFactory;
+
+import javax.management.AttributeNotFoundException;
+import javax.management.InstanceNotFoundException;
+import javax.management.MBeanException;
+import javax.management.MBeanServer;
+import javax.management.MalformedObjectNameException;
+import javax.management.ObjectName;
+import javax.management.ReflectionException;
+
+import com.hazelcast.core.Hazelcast;
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+
+public class Cluster {
+
+    public static void main(String[] args) throws Exception {
+        System.setProperty("hazelcast.jmx", "true");
+
+        Hazelcast.newHazelcastInstance();
+        Hazelcast.newHazelcastInstance();
+        Hazelcast.newHazelcastInstance();
+        HazelcastInstance hz = Hazelcast.newHazelcastInstance();
+        System.out.println("Hazelcast cluster was started. Number of members: " + hz.getCluster().getMembers().size());
+
+        IMap<String, String> map = hz.getMap("cities");
+        map.put("where to live", "Brno");
+        map.put("where to have a joy", "Paris");
+        map.put("where to go next", "London");
+        System.out.println("Entries were added to 'cities' IMap, current size is: " + map.size());
+
+        // time-to-live is set on the 'cities' map, let's do some statistics over the time
+        for (int i = 0; i < 15; i++) {
+            Thread.sleep(2000L);
```

Comment:
You can use `com.hazelcast.examples.helper.CommonUtils.sleepSeconds` here, which makes it a bit easier to read (and maybe you also don't need the `throws Exception` then).

Reply:
OK, thanks for looking into this. I'll fix it.